pkg/minikube/download/image.go

```diff
@@ -43,10 +43,16 @@ var (
 	}
 )
 
-// imageExistsInCache if img exist in local cache directory
-func imageExistsInCache(img string) bool {
+// imagePathInCache returns path in local cache directory
+func imagePathInCache(img string) string {
 	f := filepath.Join(constants.KICCacheDir, path.Base(img)+".tar")
 	f = localpath.SanitizeCacheDir(f)
+	return f
+}
+
+// ImageExistsInCache if img exist in local cache directory
+func ImageExistsInCache(img string) bool {
```

Comment:
I dont see this func be used outside `download` paockage any reason it is public ?
and also how about we move it to `image` package `image.ExistsInCache(img)` reads much better  than `download.ImageExistsInCache`

wdyt @afbjorklund

Reply:
https://github.com/kubernetes/minikube/pull/11346#issuecomment-841095916